### PR TITLE
Make footer stick to bottom of screen

### PR DESCRIPTION
**Pull request by appleJax**
# What's in this PR?

Makes footer stick to bottom of page.

### CODE REVIEW

File: app/styles/main.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@ html, body {
   height: 100%;
 }
 
+.body-container {
+  min-height: calc(100vh - 317px);
```
Comment (JoshSmith) [inline app/styles/main.scss]: What happens when the footer's height changes? Like when we make it responsive?

Comment (appleJax) [inline app/styles/main.scss]: We'd add a media query to change the pixel subtraction at the same breakpoint that the footer height changes.

Comment (appleJax) [inline app/styles/main.scss]: We could even put the nav and footer heights into a variable to make the "magic number" more clear.

Comment (JoshSmith) [inline app/styles/main.scss]: Part of what I'm getting at is that if the footer height changes in the future (someone adds another social link, etc) then we end up having to modify this CSS to make it not break.

I think maybe the complexity is higher than the benefit.

Thoughts?

Comment (appleJax) [inline app/styles/main.scss]: Yeah, good point.

Comment (JoshSmith) [inline app/styles/main.scss]: My real fear is just maintainability vs value. A sticky footer is a nice to have, but maintainable code is a must-have.

Anyway, that's why I'm glad to have seen the PR so we can see what the tradeoff looks like. I think the tradeoff is too much. Would you agree?

Comment (appleJax) [inline app/styles/main.scss]: Yeah, I agree. This solution is not maintainable. I'll keep the sticky footer in the back off my mind though, maybe there's a better way to do it.

**Comment by JoshSmith** [comment]
Closing due to discussion above.

**Pull request closed without merging**